app/components/explore-side-bar.js

```diff
@@ -0,0 +1,11 @@
+import Ember from 'ember';
+import { eventTypes, eventTopics } from 'open-event-frontend/utils/dictionary/event';
+
+const { Component } = Ember;
+
+export default Component.extend({
+  willRender() {
+    this.set('event-types', eventTypes);
```

Comment:
Don't do it like this. See how it is done in `components/forms/wizard/basic-details-step.js:L178`. Please implement like that.